bin/compile

```diff
@@ -71,13 +71,22 @@ else
   minecraft_url="https://s3.amazonaws.com/Minecraft.Download/versions/${minecraft_version}/minecraft_server.${minecraft_version}.jar"
 fi
 
-echo -n "-----> Installing Minecraft ${minecraft_version}... "
+if [[ -z "${MINECRAFT_SERVER_URL}" ]]; then
+  echo "No custom server URL set..."
+else
+  echo "-----> Setting custom download URL..." 
+  minecraft_url="${MINECRAFT_SERVER_URL}"
+  echo "-----> Set custom download URL to ${minecraft_url}" 
+fi
+
+echo "-----> Installing Minecraft ${minecraft_version}... "
+echo "Where URL = ${minecraft_url}"
 curl -o minecraft.jar -s -L $minecraft_url
 mv minecraft.jar $BUILD_DIR/minecraft.jar
 echo "done"
 
 if [ -n "${MINECRAFT_EULA:-""}" ]; then
-  echo -n "-----> Accepting Minecraft EULA... "
+  echo "-----> Accepting Minecraft EULA... "
```

Comment:
was the `-n` causing problems? I know it might not work on different versions of Bash, but it should work on heroku